react-components/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "lint": "eslint 'src/**/*.js'",
     "webpack:dev": "webpack --mode=development",
-    "webpack:prod": "webpack --mode=production",
+    "webpack:prod": "webpack --mode=production && node css-template.js",
```

Comment:
I'm not sure what you exactly want to achieve: only generate the files when we release (but they wouldn't be committed in the repo, right?) so that they are included in the NPM package? (sorry I haven't checked the recent changes to create the NPM package).

Reply:
The templates are created in the `dist` folder which is published as npm package as part of the release. It's also generated for normal builds, but since the `dist` folder is part of `.gitignore`, it will never be committed.

Reply:
Aren't "normal builds" building with `webpack:dev`? Also if we do that for React Components, do we also want to the same for the `css` styles of the HTL components?

Reply:
> Aren't "normal builds" building with webpack:dev?
Correct :)

> Also if we do that for React Components, do we also want to do the same for the css styles of the HTL components?
Those files already exist in https://github.com/adobe/aem-cif-guides-venia/tree/main/ui.frontend/src/main/styles. We could automate the generation of those files too, but it would work differently, since we would need to commit them directly to the venia project + archetype.